src/server/multi_test.cc

```diff
@@ -861,4 +862,21 @@ TEST_F(MultiEvalTest, MultiSomeEval) {
   EXPECT_THAT(brpop_resp, ArgType(RespExpr::NIL_ARRAY));
 }
 
+TEST_F(MultiEvalTest, ScriptSquashingUknownCmd) {
+  absl::FlagSaver fs;
+  absl::SetFlag(&FLAGS_lua_auto_async, true);
+
+  // The script should abort only at the second error, no further commands should be executed
```

Comment:
Why abort at the second and not first?

Reply:
Didn't add clarifications that the first is `pcall` and the second is `call`, I'll add them